core/src/main/java/org/eclipse/hono/util/CreateResult.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2019 Contributors to the Eclipse Foundation
```

Comment:
this is a new class, so it should only be 2019

Reply:
fixed.